packages/cli/src/profiles.ts

```diff
@@ -0,0 +1,29 @@
+import type { RenderOptions } from "./render/index.js";
+
+type Profile = Pick<Required<RenderOptions>, "ignoreResolutions">;
+
+export const profiles = {
+  strict: {
+    ignoreResolutions: [],
+  },
+  node16: {
+    ignoreResolutions: ["node10"],
+  },
+  "esm-only": {
+    ignoreResolutions: ["node10", "node16-cjs"],
+  },
+  "node16-only": {
```

Comment:
Can we remove this one? I can’t think of a good reason why something should pass `node16` but fail `bundler`.